rvepak/mailroom.py

```diff
@@ -0,0 +1,137 @@
+import os
+import sys
+import prettytable
+
+donors = [('Jack', 1), ('John', 2), ('Ray', 3), ('Kelly', 2), ('Rick', 1)]
+
+maxTriesCount = 0
+
+
+def inputPrompts():
+    '''
+    this method prompts for the 3 values
+    '''
+    global maxTriesCount
+    allowedResponsesList = allowedResponses(allowedResponsesMap)
+    response = input('Choose an action from the following {}: '.format(
+        allowedResponsesList))
+    if str(response) not in allowedResponsesList:
+        while maxTriesCount < 3:
+            print(
+                'Response {} is not allowed. Allowed responses are {}. Number of trials left = {}. Please try again.'.
+                format(
+                    str(response), allowedResponsesList, str(3 -
+                                                             maxTriesCount)))
+            maxTriesCount += 1
+            inputPrompts()
+    else:
+        returnRoutineToBeCalled(allowedResponsesMap, response)()
+
+
+def returnRoutineToBeCalled(allowedResponsesMap, response):
+    '''
+    this method returns the function to be called based on the input response
+    '''
+    for tpl in allowedResponsesMap:
```

Comment:
Maybe use something more descriptive that `tpl` for a variable name.  I'm guessing you are referring to tuple, but not sure.  Typical variable name in a for loop is `item` if you don't want to give it clear descriptor.